examples/kubernetes/hpe-nimble-storage/README.md

```diff
@@ -62,12 +62,12 @@ Cloning supports two modes of cloning. Either use `cloneOf` and reference a PVC
 | createSnapshot | Boolean | Indicates that a new snapshot of the volume should be taken matching the name provided in the `snapshot` parameter. If the `snapshot` parameter is not specified, a default name will be created. |
 
 ### Import parameters
-Importing volumes to Kubernetes requires the source Nimble volume to be offline. All previous Access Control Records and Initiator Groups will be stripped from the volume when put under control of the HPE CSI Driver.
+Importing volumes to Kubernetes requires the source Nimble volume to be offline. In case of reverse replication, the upstream volume should not have any active sessions and to be in offline state. All previous Access Control Records and Initiator Groups will be stripped from the volume when put under control of the HPE CSI Driver.
```

Comment:
Doesn't the volume being offline imply that there are no active sessions?  I haven't seen an online volume with active sessions before.

Reply:
Agreed. I will update the change to:
"In case of reverse replication, the upstream volume should be in offline state"